tests/app/views/test_status.py

```diff
@@ -13,6 +13,22 @@ def test_status(self):
 
             assert response.status_code == 200
 
+    def test_status_returns_json(self):
+        with self.app.app_context():
+            response = self.client.get('/_status')
+
+            assert response.json
+
+    def test_status_includes_meta_information(self):
+        with self.app.app_context():
+            response = self.client.get('/_status')
+
+            for index, details in response.json["es_status"].items():
+                if not index.startswith("."):
+                    assert details["mapping_generated_from_framework"]
```

Comment:
I guess these kind of assertions are just checking for a non-`None` value?

Reply:
It checks for a truthy value, so not `None`, 0, an empty list or dict etc.

Reply:
Testing for truthiness; almost all values in Python can be treated as a `bool`, and most will be true with a few exceptions https://docs.python.org/3/library/stdtypes.html#truth-value-testing